src/WSEvents.cpp

```diff
@@ -911,6 +915,38 @@ void WSEvents::OnSceneItemVisibilityChanged(void* param, calldata_t* data) {
 	instance->broadcastUpdate("SceneItemVisibilityChanged", fields);
 }
 
+/**
+ * An item's transfrom has been changed.
+ *
+ * @return {String} `scene-name` Name of the scene.
+ * @return {String} `item-name` Name of the item in the scene.
+ *
+ * @api events
+ * @name SourceItemChanged
+ * @category sources
+ * @since 4.5.1
+ */
+void WSEvents::OnSceneItemTransform(void* param, calldata_t* data) {
+	WSEvents* instance = static_cast<WSEvents*>(param);
+
+	obs_scene_t* scene = nullptr;
+	calldata_get_ptr(data, "scene", &scene);
+
+	obs_sceneitem_t* sceneItem = nullptr;
+	calldata_get_ptr(data, "item", &sceneItem);
+
+	const char* sceneName =
+		obs_source_get_name(obs_scene_get_source(scene));
+	const char* sceneItemName =
+		obs_source_get_name(obs_sceneitem_get_source(sceneItem));
+
+	OBSDataAutoRelease fields = obs_data_create();
+	obs_data_set_string(fields, "scene-name", sceneName);
+	obs_data_set_string(fields, "item-name", sceneItemName);
+
```

Comment:
Please add the transform state in the event properties. You can reuse the schema used in the `GetSceneItemProperties` RPC method.

Reply:
That is of course unless it doesn't trigger when you transform a non active scene...

Reply:
@BarryCarlyon It's fine, still good to have.